foundationsync/indico_foundationsync/blueprint.py

```diff
@@ -0,0 +1,30 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2019 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+from __future__ import unicode_literals
+
+from flask import redirect
+from werkzeug.exceptions import NotFound
+
+from indico.core.plugins import IndicoPluginBlueprint
+from indico.modules.rb.models.rooms import Room
+
+
+blueprint = IndicoPluginBlueprint('foundationsync', 'indico_foundationsync')
+
+
+@blueprint.route('!/rooms/resolve/<building>/<floor>-<number>')
+def resolve_room(building, floor, number):
+    room = Room.query.filter(
+        ~Room.is_deleted,
+        Room.building == building,
+        Room.floor == floor,
+        Room.number == number
+    ).one_or_none()
+    if not room:
+        raise NotFound
```

Comment:
There's also `first_or_404()` (not sure if there's `one_or_404()` as well)

Reply:
Nope. And I want it to error if there's more than one result.